app/setup/get.controller.js

```diff
@@ -7,11 +7,11 @@ const _ = require('lodash')
 const {getSessionVariable} = require('../../common/config/cookies')
 
 module.exports = (req, res) => {
-  // todo show different page with different params depending on mandate type
   const mandate = res.locals.mandate
   const session = getSessionVariable(req, mandate.externalId)
   const params = {
     mandateExternalId: mandate.externalId,
+    mandateType: mandate.type,
     description: mandate.transaction.description,
```

Comment:
it is a bit counter-intuitive that transaction is set to {}, I would _probably_ have liked to see the change in the controller itself, but it can be fine, I guess